budget/web.py

```diff
@@ -13,6 +13,8 @@
     render_template, request, session, url_for, send_file
 from flask_mail import Mail, Message
 from flask_babel import get_locale, gettext as _
+from werkzeug.security import generate_password_hash, \
```

Comment:
It might better to make another PR to remove all the backslashes instead of using 2 different syntaxes ?